### PR TITLE
Feat/cofounder profile

### PR DESCRIPTION
**Pull request by SupriyaShekar123**
# What does this pull request do?

In this pull request, small changes are made in cofounder profile video page and  cofounder personality test. Also made a new component Cofounder welcome page. It has information on how to complete the registration process. Added profile form, where the cofounder answers some questions. All the components are linked.

### CODE REVIEW

File: src/components/Cofounder/CofounderStart.js [hunk 1]
```diff
@@ -3,132 +3,149 @@ import { css, Global, jsx } from '@emotion/core';
...
-        } else {
-            setUiState('displayingLogin');
-        }
-    };
-    const closeRegistration = () => {
-        setUiState('notDisplayingLogin');
-    };
+	const existingUser = () => {
+		props.history.push('/Cofounder/login');
+		console.log('props', props);
```
Comment (Emnetdegafe) [inline src/components/Cofounder/CofounderStart.js:14]: There are some logs that we need to remove.

File: src/components/Cofounder/CofounderDashboard.js [hunk 2]
```diff
@@ -1,59 +1,514 @@
...
-              <div className="assess-tiles">
-                <img alt="icon" className="icons" src={submitIdea}></img>
-                <h4>Submit your first idea</h4>
-              </div>
-            </Link>
-          </div> */}
-        </div>
-      </div>
-    );
+	const [ user, setUserData ] = useState({});
```
Comment (Emnetdegafe) [inline src/components/Cofounder/CofounderDashboard.js:12]: I understand this is now in a new component.

**Review by Emnetdegafe — APPROVED** [review]
Some console logs, except for that, it looks good to me.

File: src/components/Cofounder/CofounderPersonalityTest.js [hunk 3]
```diff
@@ -1,241 +1,257 @@
...
-  )
+	const [ testResult, setTestResult ] = useState({ personalityTest: '' });
+	const [ errorCheck, seterrorCheck ] = useState('false');
+	const [ successMsg, setSuccessMsg ] = useState(false);
+
+	const handelSubmit = async (e) => {
+		e.preventDefault();
+		setSuccessMsg(true);
+
+		const response = await fetch(`${baseUrl}/users`, {
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderPersonalityTest.js:17]: We do not use the fetch API, please use request, same as in the rest of the app :)

**Review by dashakrolik — CHANGES_REQUESTED** [review]

File: src/components/Cofounder/CofounderPersonalityTest.js (hunk 3 above)
Comment (SupriyaShekar123) [reply src/components/Cofounder/CofounderPersonalityTest.js:17]: Actually my prettier is not working so had indentation problems, so Reinko Pulled my branch and formated my code and made a new pull request. So we do not want to merge this branch. I have informed Reinko. He will be making the changes.

Comment (SupriyaShekar123) [reply src/components/Cofounder/CofounderPersonalityTest.js]: I have made the request using super agent(same as in the rest of the app :)). I have removed all the console logs. Please review the code.

**Review by DannyS-stack — APPROVED** [review]

**Pull request merged**